### DISCUSSION TITLE
Plots of data

### DISCUSSION THREAD

**Discussion by bbartling** [joyja/tentacle-plc#28] [Q&A]
Can tentacle make simple line plots of data?

Curious if the rest API is fully functional as well?

**Comment by joyja** [comment]
Hi Ben, it doesn't do it on its own atm, but I think think this is a great feature request for the front end. I also have a project called mantle (https://github.com/joyja/mantle) that listens to a broker for MQTT Sparkplug B and historizes to postgresql (preferable with the Timescale DB add-on). Mantle doesn't do a simple line chart either, but it does expose historical data to external apps via a graphql api.

On the REST API front, I've been using it from within the individual tasks by importing fetch doing something like this:
```
const fetch = require('node-fetch')

module.exports = async function ({ global }) {
    global.apiVersion = await fetch('http://host/api/version').then((response) => {
        return response.json()
    }).catch((error) => {
        console.error(error)
    })
}
```

I don't have it tied into variables natively yet, but I still agree this should be on the roadmap as well.
